pages/advanced/erc-7579/tutorials/7579-tutorial.mdx

```diff
@@ -0,0 +1,194 @@
+# **How to build an app with Safe and 7579**
+
+The smart account ecosystem was fragmented, with each provider building its own modules often incompatible with other smart account implementations. Developers had to build new modules compatible with their smart accounts or miss out on essential application features.
+
+[ERC-7579](https://docs.safe.global/advanced/erc-7579/overview) aims to ensure interoperability across implementations. It defines the account interface so developers can implement modules for all smart accounts that follow this standard. The Safe7579 Adapter makes your Safe compatible with any 7579 modules. As a developer building with Safe, you get access to a rich ecosystem of modules to make your application feature-rich.
+
+Let's say you want to build an app to enable scheduling transfers for monthly salaries to a team of contributors. However, Safe does not offer a native module for scheduling transfers. With the 7579 compatibility, you can use Rhinestone's [Scheduled Transfer module](https://github.com/rhinestonewtf/modulekit/blob/examples/audit-ready/examples/src/ScheduledTransfers/ScheduledTransfers.sol) with Safe to build an app to schedule transfers ahead of time.
+
+This tutorial will teach you to build an app that can:
+
+- Deploy a 7579-enabled Safe.
+- Create a scheduled transaction.
+- Execute it at the requested date and time.
+
+![screenshot](../../../../assets/7579-tutorial.png)
+
+## What you’ll need
+
+**Prerequisite knowledge:** You will need some basic experience with [React](https://react.dev/learn), [Next.js](https://nextjs.org/docs), [ERC-4337](https://docs.safe.global/home/4337-overview) and ERC-7579.
+
+Before progressing with the tutorial, please make sure you have:
+
+- Downloaded and installed [Node.js](https://nodejs.org/en/download/package-manager) and [pnpm](https://pnpm.io/installation).
+- Created an API key from [Pimlico](https://www.pimlico.io/).
+
+**Note:** If you wish to follow along using the completed project, you can [check out the GitHub repository](https://github.com/5afe/safe-7579-tutorial) for this tutorial.
```

Comment:
this looks like a wrong link/private repo fyi

Reply:
@louis-md can we make this public - should be fine to already do that :)